src/main/java/com/photolocos/enterprise/service/IPhotoService.java

```diff
@@ -8,15 +8,15 @@
 
 public interface IPhotoService {
 
-    PhotoDTO fetchByLocation(String location);
+    Set<PhotoDTO> fetchByArea(String area) throws Exception;
 
```

Comment:
Believe this should be fetchByArea. Multiple photos could be placed within a given area so this returns a set of photos